styles/style.css

```diff
@@ -95,7 +97,21 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+.rect {
```

Comment:
Careful with classnames that aren't very descriptive. If we see this classname in the document it's difficult to determine the elements it is being used on, and then it's more difficult to make changes to the class because we aren't sure which elements will be affected. 
You've got the right idea to create some base properties in `.rect` for all buttons, and then override some of these properties in `.white`. Another way to do this more semantically might be to apply the base styles to the button element, e.g.
```
button {
    background-color: var(--orange-dark);
    color: var(--white);
    border: none;
    border-radius: 4px;
    padding: 5px 12px;
    margin-left: 32px;
}
```
and then create a modifier class for the white button, e.g.
```
.button--secondary {
    background-color: var(--white);
    color: var(--orange-dark);
    margin-left: 0;
}
```
This class can then be used on the white button, if we assume the orange colour is the primary colour, e.g.
```
<button class="button--secondary">Volunteer</button>
```

The `--` is just a naming convention, but semantically we can assume that the new class is used on button elements and will share the same base styles applied to button :)